src/sections/HomeStats/index.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Section from "root/components/Section";
+import Themes from "root/components/Themes";
+import Stats from "root/components/Stats";
+import Title from "root/components/Title";
+import BigText from "root/components/BigText";
+import Link from "root/components/Link";
+
```

Comment:
IMHO, we could sort the imports. As with the others comments about sorting, feel free to disregard this

Reply:
Yes